### PR TITLE
Reduce debug level logs 

### PR DESCRIPTION
**Pull request by i-oden**
## **1. This PR contains the following changes...**

Reducing the number of log statements since the log files seem to become very big. Changed the logger to "dds_cli" in `__main__.py` which avoids the root logger and therefore we won't have logging from all installed and imported packages. Also reduced a few lines of logging.

## **2. The following additional changes are required for this to work**

_X_

## **3. The PR fixes the following GitHub issue / Jira task**

- [ ] GitHub issue (link):
- [X] Jira task (ID, `DDS-xxxx`): DDS-1398
- [ ] The PR does not fix a specific GitHub issue or Jira task

## **4. What _type of change(s)_ does the PR contain?**

- [ ] New feature
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [ ] Non-breaking
- [x] Bug fix
  - [ ] Breaking: _Please describe the reason for the break and how we can fix it._
  - [x] Non-breaking
- [ ] Security Alert fix
- [x] Documentation
- [ ] Tests **(only)**
- [ ] Workflow

### **Always**

- [Changelog](../CHANGELOG.md)
  - [x] Added
  - [ ] Not needed (E.g. PR contains _only_ tests)
- Rebase / Update / Merge _from_ base branch (the branch from which the current is forked)
  - [ ] Done
  - [ ] Not needed
- Blocking PRs
  - [ ] Merged
  - [x] No blocking PRs
- PR to `master` branch
  - [ ] Yes: Go to the section [PR to master](#pr-to-master)
  - [x] No

### If PR consists of **code change(s)**

- Self review
  - [x] Done
- Comments, docstrings, etc
  - [x] Added / Updated
- Documentation
  - [ ] Updated
  - [x] Update not needed

## **6. Actions / Scans**

- **Black**: Python code formatter. Does not execute. Only tests.
  Run `black .` locally to execute formatting.
  - [x] Passed
- **Prettier**: General code formatter. Our use case: MD and yaml mainly.
  Run `npx prettier --write .` locally to execute formatting.
  - [x] Passed
- **Yamllint**: Linting of yaml files.
  - [x] Passed
- **Tests**: Pytest to verify that functionality works as expected.
  - [ ] New tests added
  - [x] No new tests
  - [x] Passed
- **TestPyPi**: Build CLI and publish to TestPyPi in order to verify before release.
  - [x] Passed
- **CodeQL**: Scan for security vulnerabilities, bugs, errors
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Trivy**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed
- **Snyk**: Security scanner
  - [ ] New alerts: _Go through them and either fix, dismiss och ignore. Add reasoning in items below._
  - [ ] Alerts fixed: _What?_
  - [ ] Alerts ignored / dismissed: _Why?_
  - [x] Passed

### CODE REVIEW

File: dds_cli/file_handler_remote.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 # START LOGGING CONFIG ################################# START LOGGING CONFIG #
 ###############################################################################
 
-LOG = logging.getLogger(__name__)
+LOG = logging.getLogger(z)
```
Comment (i-oden) [inline dds_cli/file_handler_remote.py:23]:
```suggestion
LOG = logging.getLogger(__name__)
```

File: dds_cli/file_handler_remote.py [hunk 2]
```diff
@@ -115,11 +115,10 @@ def __collect_file_info_remote(self, all_paths, token):
             }
             for x, y in files.items()
         }
-        LOG.debug(f"Data (files):\n {data}")
+        # LOG.debug(f"Data (files):\n {data}")
```
Comment (i-oden) [inline dds_cli/file_handler_remote.py:118]:
```suggestion
```

**Comment by valyo** [comment]
In the local testing I still see all the `DEBUG` lines in the log file:
running:
`dds --log-file dds-auth-log.log -vv ls`
produces
```
[2023-02-10 08:57:30] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
[2023-02-10 08:57:30] dds_cli.user    70    [DEBUG  ]  Retrieving token...
[2023-02-10 08:57:30] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
[2023-02-10 08:57:30] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
[2023-02-10 08:57:30] dds_cli.user    243   [DEBUG  ]  Token file /root/.dds_cli_token does not exist.
[2023-02-10 08:57:30] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
[2023-02-10 08:57:30] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
```
in the log file
and running
`dds --log-file dds-auth-log.log ls`
produces the same thing:
```
[2023-02-10 10:05:39] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
[2023-02-10 10:05:39] dds_cli.user    70    [DEBUG  ]  Retrieving token...
[2023-02-10 10:05:39] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
[2023-02-10 10:05:39] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
[2023-02-10 10:05:39] dds_cli.user    243   [DEBUG  ]  Token file /Users/valge939/.dds_cli_token does not exist.
[2023-02-10 10:05:39] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
[2023-02-10 10:05:39] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
```
The goal was to get rid of `DEBUG` lines when not running in debug level, or?!

**Review by valyo — COMMENTED** [review]
In the local testing I still see all the `DEBUG` lines in the log file:
running:
`dds --log-file dds-auth-log.log -vv ls`
produces
```
[2023-02-10 08:57:30] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
[2023-02-10 08:57:30] dds_cli.user    70    [DEBUG  ]  Retrieving token...
[2023-02-10 08:57:30] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
[2023-02-10 08:57:30] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
[2023-02-10 08:57:30] dds_cli.user    243   [DEBUG  ]  Token file /root/.dds_cli_token does not exist.
[2023-02-10 08:57:30] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
[2023-02-10 08:57:30] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
```
in the log file
and running
`dds --log-file dds-auth-log.log ls`
produces the same thing:
```
[2023-02-10 10:05:39] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
[2023-02-10 10:05:39] dds_cli.user    70    [DEBUG  ]  Retrieving token...
[2023-02-10 10:05:39] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
[2023-02-10 10:05:39] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
[2023-02-10 10:05:39] dds_cli.user    243   [DEBUG  ]  Token file /Users/valge939/.dds_cli_token does not exist.
[2023-02-10 10:05:39] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
[2023-02-10 10:05:39] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
```
The goal was to get rid of `DEBUG` lines when not running in debug level, or?!

**Comment by i-oden** [comment]
> In the local testing I still see all the `DEBUG` lines in the log file: running: `dds --log-file dds-auth-log.log -vv ls` produces
> 
> ```
> [2023-02-10 08:57:30] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
> [2023-02-10 08:57:30] dds_cli.user    70    [DEBUG  ]  Retrieving token...
> [2023-02-10 08:57:30] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
> [2023-02-10 08:57:30] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
> [2023-02-10 08:57:30] dds_cli.user    243   [DEBUG  ]  Token file /root/.dds_cli_token does not exist.
> [2023-02-10 08:57:30] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
> [2023-02-10 08:57:30] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
> ```
> 
> in the log file and running `dds --log-file dds-auth-log.log ls` produces the same thing:
> 
> ```
> [2023-02-10 10:05:39] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
> [2023-02-10 10:05:39] dds_cli.user    70    [DEBUG  ]  Retrieving token...
> [2023-02-10 10:05:39] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
> [2023-02-10 10:05:39] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
> [2023-02-10 10:05:39] dds_cli.user    243   [DEBUG  ]  Token file /Users/valge939/.dds_cli_token does not exist.
> [2023-02-10 10:05:39] dds_cli.user    82    [INFO   ]  No saved token found, or token has expired, proceeding with authentication
> [2023-02-10 10:05:39] dds_cli.user    92    [DEBUG  ]  Starting authentication on the API...
> ```
> 
> The goal was to get rid of `DEBUG` lines when not running in debug level, or?!

Not get rid of, just reduce the number of them. 
If you change the `__main__.py` `getLogger` part from
```python
LOG = logging.getLogger("dds_cli")
```
to 
```python
LOG = logging.getLogger()
```

You'll notice a difference. 
And I've removed some debugging that was not needed. We still need the debugging though, but not to the extent that we were before.

**Comment by valyo** [comment]
well, I meant that I get the same thing in this branch and in `dev`, i.e. with both
`LOG = logging.getLogger("dds_cli")`
and
`LOG = logging.getLogger()`

**Comment by i-oden** [comment]
@valyo The logging to the file is always in debug level

**Comment by i-oden** [comment]
> well, I meant that I get the same thing in this branch and in `dev`, i.e. with both `LOG = logging.getLogger("dds_cli")` and `LOG = logging.getLogger()`

That's strange... 
I get this
```
[2023-02-10 10:35:26] dds_cli.base    74    [DEBUG  ]  Attempted operation: ls
[2023-02-10 10:35:26] dds_cli.user    70    [DEBUG  ]  Retrieving token...
[2023-02-10 10:35:26] dds_cli.user    74    [DEBUG  ]  Checking if token file exists.
[2023-02-10 10:35:26] dds_cli.user    240   [DEBUG  ]  Attempting to retrieve token from file...
[2023-02-10 10:35:26] dds_cli.user    258   [DEBUG  ]  Token retrieved from file.
[2023-02-10 10:35:26] urllib3.connectionpool 1003  [DEBUG  ]  Starting new HTTPS connection (1): delivery.scilifelab.se:443
[2023-02-10 10:35:35] urllib3.connectionpool 456   [DEBUG  ]  https://delivery.scilifelab.se:443 "GET /api/v1/proj/list HTTP/1.1" 200 39522
```

**Comment by valyo** [comment]
> But anyway, there were some debug logging where we were iterating through a lot of lines and logging them all. I've removed those too. So the level of debug level logs in the file should be reduced either way

then I can just approve

**Review by valyo — APPROVED** [review]

**Pull request merged**